components/style/inline.rs

```diff
@@ -0,0 +1,44 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use computed_values::white_space;
+
+pub trait WhitespaceMethods {
```

Comment:
Since the type is defined in the same crate, I don’t think a trait is necessary here. Have you tried making this an inherent `impl white_space::T {}`?

Less important: I’m not sure about this file’s name or having a separate file in the first place. Do you expect to have more code related to inline layout go here in future?